history.js

```diff
@@ -499,6 +499,22 @@
     return !!historyPushState;
   }
 
+  /**
+   * This method attempts to bind a function to global.
+   *
+   * @param {Function} [func] The function to be bound
+   * @return {Function} Returns the bound function or func
+   */
+  function maybeBindToWindow(func) {
+    if (window &&
+        window.EventTarget &&
+        typeof window.EventTarget.prototype.addEventListener === 'function' &&
```

Comment:
Will this work in IE8?

I also need to support IE8

Reply:
`window.EventTarget` doesn't appear in a microsoft browser until edge (not even IE11!)